### PR TITLE
all items done. should be verifed

### PR DESCRIPTION
**Pull request by Nick-Shcherban**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,15 +1,87 @@
...
+                   <span class="title2">Website</span>
+                </div> 
+                <div class="rectangle_basic">
+                    <span class="price-card_price">99€</span>
+                </div>
+
+                <div class="price-card_description"><span>Simple website made in the world's most popular CMS system, Wordpress.</span></div>     
+                
+                <ul class="price-card_options-list">
+                    <li class="list-item dark"><img src="Shape 1.svg"><span>Website in WordPress</span> </li>
```
Comment (Nazar0n) [inline index.html]: If you can don't use space in file names

File: index.html [hunk 2]
```diff
@@ -1,15 +1,87 @@
...
+                    <li class="list-item"><img src="Shape 1.svg"><span>3 sub pages</span></li>
```
Comment (Nazar0n) [inline index.html]: Use "./" in `src`. For example `src="./Shape.svg"`

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+    font-weight: 700;
+}
+.price-card_description { 
+    display: flex;
+    height: 116px;
+    justify-content: center;
+    align-items: center;
+}    
+
+.price-card_description > span {
```
Comment (Nazar0n) [inline styles.css]: Maybe use classes instead `>`

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+.price-card_options-list {
+    display: flex;
+    flex-direction: column;
+    width: 380px;
+    height: 300px;
+    top: 351px;
+    padding: 0;   
+    list-style: none;
+}
+img {
```
Comment (Nazar0n) [inline styles.css]: Use classes

File: index.html (hunk 2 above)
Comment (Nick-Shcherban) [reply index.html]: done in the next commit

File: styles.css (hunk 3 above)
Comment (Nick-Shcherban) [reply styles.css]: is it necessary to assign class for only one child element?

File: styles.css (hunk 4 above)
Comment (Nick-Shcherban) [reply styles.css]: done in the next commit

File: index.html [hunk 5]
```diff
@@ -1,15 +1,87 @@
...
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=<>, initial-scale=1.0">
     <title>Document</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
+        integrity="sha256-l85OmPOjvil/SOvVt3HnSSjzF1TUMyT9eV0c2BzEGzU=" crossorigin="anonymous" />
+       <link rel="stylesheet" href="styles.css">
```
Comment (velmyk) [inline index.html]: Keep formatting and spacing consistent. `ctrl+alt+f` in VS Code should do the thing

File: index.html [hunk 6]
```diff
@@ -1,15 +1,87 @@
...
+                   <span class="title2">Website</span>
+                </div> 
+                <div class="rectangle_basic">
+                    <span class="price-card_price">99€</span>
+                </div>
+
+                <div class="price-card_description"><span>Simple website made in the world's most popular CMS system, Wordpress.</span></div>     
+                
+                <ul class="price-card_options-list">
+                    <li class="list-item dark"><img class="tick" src="./Shape 1.svg"><span>Website in WordPress</span> </li>
```
Comment (velmyk) [inline index.html]: Good solution with extra class.

Another possible solution id pseudo classes:
```
.list-item:nth-child(odd) {
   background-color: some-different-color;
}
```

File: index.html [hunk 7]
```diff
@@ -1,15 +1,87 @@
...
-    <span>Hello World!</span>
-    <span>It doesn't work</span>
-    <span>check</span>
-    <span> THIS IS UPDATED TEXT </span>
-</body>
+
+    <body>
+            <div class="basic">
+                <div class="price-card_title">
+                   <span class="title1">Basic</span>
```
Comment (velmyk) [inline index.html]: Try to avoid classes names with numbers:
`title` and `subtitle` looks better for me

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,139 @@
+.basic, .business, .webshop {
```
Comment (velmyk) [inline styles.css]: Do you really need 3 classes?
Can it be just one?

File: styles.css (hunk 3 above)
Comment (velmyk) [reply styles.css]: Yes. Better to use extra classes instead of nested / complex selectors

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+    opacity: 50%;   
+    color: #FFFFFF;
+    font-weight: 300;
+    font-size: 15px;
+    line-height: 50px;
+}
+.dark {
+    background-color: #2A3438
+}
+.button_basic, .button_business, .button_webshop {
```
Comment (velmyk) [inline styles.css]: Can it be just one class?

**Review by velmyk — APPROVED** [review]

File: styles.css (hunk 3 above)
Comment (Nick-Shcherban) [reply styles.css]: done

File: index.html (hunk 5 above)
Comment (Nick-Shcherban) [reply index.html]: done in the last commit

File: index.html (hunk 7 above)
Comment (Nick-Shcherban) [reply index.html]: done in the last commit

File: styles.css (hunk 8 above)
Comment (Nick-Shcherban) [reply styles.css]: done in the last commit

File: index.html (hunk 7 above)
Comment (Nick-Shcherban) [reply index.html:16]: done in the last commit

File: styles.css (hunk 9 above)
Comment (Nick-Shcherban) [reply styles.css]: for sure. done in the last commit.

**Review by velmyk — APPROVED** [review]

**Pull request merged**